odoo/models.py

```diff
@@ -227,6 +227,7 @@ class BaseModel(object):
 
     _name = None                # the model name
     _description = None         # the model's informal name
+    _is_business = False        # should be True for important business models
```

Comment:
What about "main model in application"?
Odoo is an application framework at its core.
"Business" is very ERP-oriented, "Application" is more general purpose.
The model `ir.model` has no direct logic for a regular user: it is fundamentally a technical artifact.